integration-cli/docker_api_attach_test.go

```diff
@@ -96,90 +97,90 @@ func (s *DockerSuite) TestGetContainersWsAttachContainerNotFound(c *testing.T) {
 	assert.Assert(c, strings.Contains(getErrorMessage(c, b), expected))
 }
 
-func (s *DockerSuite) TestPostContainersAttach(c *testing.T) {
-	testRequires(c, DaemonIsLinux)
+func (s *DockerSuite) TestPostContainersAttach(t *testing.T) {
```

Comment:
Looks like renaming c to t causes ci to fail (because it now detects it as a  new test)

We should indeed consider changing all of these to "t", but perhaps to unblock this PR you could rename it back ?